File: e08-emall-parent/e08-seller-api/src/main/java/cn/minture/seller/service/SpecificationService.java
package cn.minture.seller.service;

import java.util.List;
import java.util.Map;

import cn.minture.emall.entity.Specification;
import cn.minture.emall.entity.TbSpecification;
import cn.minture.result.ShopResult;

public interface SpecificationService {

	 public void add(Specification specification);
	 public void update(Specification specification);
	 public void delete(Long [] ids);
	 public Specification findById(Long id);
	 public ShopResult queryByPager(TbSpecification specification,int pageNo,int pageSize);
	 public List<TbSpecification> queryAll();
	 public List<Map>selectOptionList();
}

File: e08-emall-parent/e08-manager-web/src/main/webapp/js/service/specificationService.js
app.service("specifyService",function($http){
		this.findPage=function(page,size,searchEntity)
	{
			return $http.post('../specify/findByPager?pageNo=' + page + '&pageSize=' + size,searchEntity)
	}
    this.save=function(entity){
    	
    	return $http.post('../specify/add',entity);
    }
    this.update=function(entity){
    	return $http.post('../specify/update',entity)
    }
    
	this.findById=function(id)
	{
		return $http.get('../specify/findById?id=' + id)
	}
    this.dele=function(selectIds)
    {
    	return  $http.get('../specify/delete?ids=' +selectIds)
    }
	});